src/doc/reference/glossary/cilk.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Cilk
+tags: Cilk
+---
+A simple set of extensions to the C and C++
+programming languages that allows a
+programmer to express concurrency
+easily.
```

Comment:
We often distinguish "concurrency" from "parallelism,"  (See, for example, this CppCon presentation by Pablo Halpern, who was formerly at Cilk Arts: https://www.youtube.com/watch?v=y0GSc5fKtl8&ab_channel=CppCon.)  In light of that modern distinction, I think it's more precise to say, "to express task parallelism" (with a link to the entry on task parallel).

Reply:
Thank you. Having done this, I wonder if we want to add something to the definition of "task parallel" that specifically addresses "task parallelism"?